SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -230,8 +230,16 @@ private extension PersonController {
                          bottom: donationButtonContainerView.topAnchor, trailing: nil,
                          padding: .zero, size: .zero)
         
-        donationButtonContainerView.setButtonPressed {
-            // TODO: Donation button action
+        donationButtonContainerView.setButtonPressed { [weak self]
+            in
+            guard let self = self else { return }
+
+            var donation = self.person.donations.first
+            donation?.person = self.person
+            let donationsDetailsVC = DonationsMoreDetailsController()
+            donationsDetailsVC.donation = donation
```

Comment:
donation screen now has loading logic (just added today), what do you think about using that instead? 
otherwise this looks good!

Reply:
Sure, will do it right now.